.github/workflows/integration-tests.yml

```diff
@@ -92,6 +92,10 @@ jobs:
         if: ${{ matrix.php != 8.0 }}
         run: composer testwp --no-interaction
 
+      - name: Run integration tests (multi site)
+        if: ${{ matrix.php != 8.0 }}
+        run: composer testwp-ms --no-interaction
+
```

Comment:
Why is it not being run for PHP 8.0?

Reply:
> Why is it not being run for PHP 8.0?

I wasn't entirely sure about that -- I copied the block above. I'm not 100% sure how the coverage portion works, but it is running the Integration tests with multisite enabled.  Happy to remove the `if` line here, though.